### PR TITLE
[ADD] l10n_us_product: Product weight and volume in US units

### PR DESCRIPTION
**Pull request by lasley**
* New module that allows you to record product weight in ounces and volume in inches cubed

### CODE REVIEW

File: l10n_us_product/models/product_template.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+    @api.multi
+    def _inverse_weight_oz(self):
+        for record in self:
+            record.weight = self._convert_kilogram_ounce(
+                record.weight_oz, to_oz=False,
+            )
+
+    @api.multi
+    def _compute_volume_in(self):
```
Comment (lasley) [inline l10n_us_product/models/product_template.py]: These calculations are probably wrong m3 != m, in != m3.

Likely calculation: `convert( (cubed ** (1.0/3.0) ) ** 3`

File: l10n_us_product/models/product_template.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+    @api.multi
+    def _inverse_volume_in(self):
+        for record in self:
+            record.volume = self._convert_meter_inch(
+                record.volume_in, to_in=False,
+            )
+
+    @api.model_cr_context
+    def _convert_kilogram_ounce(self, quantity, to_oz=True):
```
Comment (lasley) [inline l10n_us_product/models/product_template.py:51]: Note this is overly complex to allow for usage in other models (such as the module I'm about to submit for stock.picking)

**Comment by max3903** [comment]
@lasley Your module is installed on Runbot but the view is not there.

Otherwise everything looks good to me.

For a future version, what do you think of adding an option to choose the default unit of measure for weight? I have customers who enters weights in pounds (lb), not ounces.

**Comment by lasley** [comment]
I deleted all the coveralls spam & will take a look at why the hell it's still commenting in this repo (so annoying IMO).

> Your module is installed on Runbot but the view is not there.

Yeah for some reason I put `view` instead of `data` in the manifest. Brain fart, fixed in e2118d9

> For a future version, what do you think of adding an option to choose the default unit of measure for weight? I have customers who enters weights in pounds (lb), not ounces.

I basically just added this. We run into an issue with the cubic meters though - do you know of an abstract way to do the conversions? For some reason I can't get the math to work by simply rooting the volume, converting the side, then squaring the converted side. I'll double check though, because I feel like that should work 😑

**Review by max3903 — APPROVED** [review]

**Comment by lasley** [comment]
Doesn't look like we have much in the way of stakeholders in this repo, so I'm just merging on @max3903 approval & a decent amount of time sitting for any dissenters.

**Pull request merged**